crypto/stark-verifier-ethereum/contracts/utils.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.6.4;
+
+
+library Utils {
+    function bit_reverse(uint64 num, uint8 number_of_bits) internal view returns (uint256 num_reversed) {
+        uint64 n = num;
+        uint64 r = 0;
+        for (uint8 k = 0; k < number_of_bits; k++) {
+            r = (r * 2) | (n % 2);
+            n = n / 2;
+        }
+        return r;
+    }
+
+    function bits_in(uint64 data) internal pure returns (uint8) {
```

Comment:
I think `num_bits` would be a better name.

Reply:
'most significant bit' is the common name for this function.